src/guide/advanced/triggers.md

```diff
@@ -138,10 +145,11 @@ this tutorial will be made obsolete in the next release.
 
 ### Event Triggers
 
-As we have said previously, all triggers are, in a sense, event triggers.
-However, this category includes the largest variety of triggers: an account
-got registered, an asset got transferred, the Queen of Hearts decided to
-burn all of her assets.
+As we have said previously, all triggers are, in a sense, event triggers
+(check out this [diagram] to understand the hierarchy.). However, this
+category includes the largest variety of triggers: an account got
```

Comment:
```suggestion
category includes the largest variety of triggers; an account got
```

Reply:
we list the types of events after the `:`, why do you suggest to change it to `;`?

Reply:
Yeah, you're right, the items are in a list, at first glance it looked like a complex series. Thanks for the clarification